### ISSUE TITLE
Webpages do not detect we are logged in properly

### ISSUE THREAD

**Issue by StefanVanDyck** [inbo/vlaams-biodiversiteitsportaal#117]
Currently our current state of authentication is not registered correctly in the portal.
This can be quite strange, for example, when downloading it will show us the input-field for submitting our e-mail adress, even though we just logged in.

Should be fixed by using a proper js library for managing our oidc token/states?
Especially need to make sure this works with the Downloads page, as requesting a download with a free-form email input field fails.
So we need to make sure we always redirect to a login form if people are not logged in, but trying to download.

**Comment by StefanVanDyck** [comment]
So I think I figured out the problem.
The main issue we had was with the Downloads page, this was the ALB rule I added to force people to be logged in when visiting the biocache downloads page.

That login did not set the "auth-cookie", which meant it did not force the Grails session to be authenticated.
([I thinks that's what this logic does](https://github.com/AtlasOfLivingAustralia/ala-security-project/blob/b8bc83d085e4252bfa7afb663cf046a4d83eb86d/ala-auth/src/main/groovy/au/org/ala/web/config/AuthPac4jPluginConfig.groovy#L253))
